Aleksandr Zorin/css/style.css

```diff
@@ -0,0 +1,120 @@
+@charset "utf-8";
+/* CSS Document */
+
+body {
+	background-image: url(../img/background.jpg);
+	background-position: center center;
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-size: cover;
+	font-family: 'Dosis', sans-serif;
+}
+
+hr {
+	color: #8E4140;
+	background-color: #8E4140;
+	height: 1px;
+}
+
+p {
+	margin: 0;
+}
+
+.weatherWidget {
+	margin-top: 100px;
+	display: flex;
+	justify-content: center;
+	flex-wrap: wrap;
+}
+
+.location {
+	background-image: url(../img/city.jpg);
+	background-size: cover;
+	width: 500px;
+	height: 750px;
+	border-radius: 10px 0 0 10px;
+	box-shadow: 0px 0px 100px 1px black;
+}
+
+.information {
+	width: 400px;
```

Comment:
@Zorin-Alex Please look at box-sizing property
https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing
You can set width to 500 and box-sizing to border-box